libs/sdk/eslint-config-skyux/package.json

```diff
@@ -21,7 +21,7 @@
     "save": "devDependencies"
   },
   "ng-update": {
-    "packageGroupName": "skyux-eslint",
+    "packageGroupName": "eslint-config-skyux",
```

Comment:
Discovered this while doing the stylelint work: using the wrong package group name.